gapic-generator/templates/default/service/client/method/def/_request_streaming.erb

```diff
@@ -4,6 +4,17 @@ unless request.is_a? ::Enumerable
   request = request.to_enum
 end
 
+<%- if method.auto_populated_fields.length > 0 -%>
+
+# Auto populate fields for this request.
+<%- end -%>
+<% method.auto_populated_fields.each do |field| %>
+if request.is_a? Hash
+  request[:<%= field %>] = SecureRandom.uuid if request.key? :<%= field %>
```

Comment:
It needs to be `if !request.key? :<%= field %>` (or probably better, `unless request.key? :<%= field %>`)

Reply:
Is there a test we could run that would verify this logic?

Reply:
Thanks! I am adding one for `echo` specifically and checking some other logic for in service/rest/test/method
/_normal.erb

Reply:
Currently `shared/test/showcase/test_helper.rb` has a comment where we need to update the version for the container image. Locally it passes by building my own binary of `gapic-showcase`. Let's coordinate next week!